config/tf_modules/aws-s3/bucket.tf

```diff
@@ -64,6 +66,61 @@ resource "aws_s3_bucket" "bucket" {
   }
 }
 
+resource "aws_cloudfront_origin_access_identity" "read" {
+  comment = "For reading bucket ${var.bucket_name}"
+}
+
+
+resource "aws_cloudfront_origin_access_identity" "read_write" {
+  comment = "For writing to bucket ${var.bucket_name}"
+}
+
+resource "aws_cloudfront_origin_access_identity" "read_write_delete" {
+  comment = "For deleting and writing to bucket ${var.bucket_name}"
+}
+
+data "aws_iam_policy_document" "s3_policy" {
```

Comment:
Can we move this to the cloudfront module? The cf module should add these perms when it gets linked to s3.

Reply:
No, we really can not because it has to live in the bucket IAM policy, which is set here.

Reply:
(and a bucket can not hold 2 policies)